### PR TITLE
Jonmv/provide filter on data plane filters

### PR DESCRIPTION
**Pull request by jonmv**
@bjorncs please review.

### CODE REVIEW

File: jdisc-security-filters/src/main/java/com/yahoo/jdisc/http/filter/security/base/JsonSecurityRequestFilterBase.java [hunk 1]
```diff
@@ -25,7 +26,7 @@
  *
  * @author bjorncs
  */
-public abstract class JsonSecurityRequestFilterBase extends AbstractComponent implements SecurityRequestFilter {
+public abstract class JsonSecurityRequestFilterBase extends ChainedComponent implements SecurityRequestFilter {
```
Comment (jonmv) [inline jdisc-security-filters/src/main/java/com/yahoo/jdisc/http/filter/security/base/JsonSecurityRequestFilterBase.java:29]: It's sad that the next person has to figure out _their_ filter isn't a child of this class, too, but I don't see a good solution here. Do you?

Comment (bjorncs) [reply jdisc-security-filters/src/main/java/com/yahoo/jdisc/http/filter/security/base/JsonSecurityRequestFilterBase.java:29]: I'm not familiar with the framework. If it do require class inheritance I don't see any quick fix.

**Review by bjorncs — DISMISSED** [review]
Nice! 👍 🎖️

File: container-core/src/main/java/com/yahoo/jdisc/http/filter/RequestFilter.java [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 
+import com.yahoo.component.chain.model.Chainable;
 import com.yahoo.jdisc.handler.ResponseHandler;
 import com.yahoo.jdisc.http.HttpRequest;
 
 /**
  * @author Einar M R Rosenvinge
  */
-public interface RequestFilter extends com.yahoo.jdisc.SharedResource, RequestFilterBase {
+public interface RequestFilter extends com.yahoo.jdisc.SharedResource, RequestFilterBase, Chainable {
```
Comment (bjorncs) [inline container-core/src/main/java/com/yahoo/jdisc/http/filter/RequestFilter.java:11]: Any reason for not having `RequestFilterBase` / `ResponseFilterBase` inherit from `Chainable` instead?

**Review by bjorncs — DISMISSED** [review]

File: container-core/src/main/java/com/yahoo/jdisc/http/filter/RequestFilter.java (hunk 2 above)
Comment (jonmv) [reply container-core/src/main/java/com/yahoo/jdisc/http/filter/RequestFilter.java:11]: Probably not.

**Pull request merged**